app/src/main/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIService.java

```diff
@@ -21,14 +21,17 @@
 public interface OpenFoodAPIService {
 
     String PRODUCT_API_COMMENT = "Official Android app";
-
+    
+    @Headers("Open Food Facts Android App")
```

Comment:
We can't use static headers like this. We need these to work for all the flavors of the app and not just OFF. Use dynamic header map instead with key "User-Agent" and value should be the app name with the version number.